common/src/test/java/org/keycloak/common/util/CollectionUtilTest.java

```diff
@@ -0,0 +1,52 @@
+package org.keycloak.common.util;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.Timeout;
+import org.keycloak.common.util.CollectionUtil;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+public class CollectionUtilTest {
+
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
+  @Rule
+  public Timeout globalTimeout = new Timeout(10000);
```

Comment:
I would remove this timeout rule and the org.junit.rules.Timeout import at line 7. 

Timeout rules are something we added in at the request of a customer to prevent any hanging tests during execution. With a small amount of tests like this, it's better for the process to cut out as much of the unnecessary stuff as possible.